### ISSUE TITLE
some SQL-example is needed for prevention of accidental upgrading to PRE-revision

### ISSUE THREAD

**Issue by antonov-impulsm** [polymorphm/pg-make-schemas#2]
some SQL-example is needed for prevention of accidental upgrading to PRE-revision.

it can be a after-insert-trigger on a revision table.

**Comment by antonov-impulsm** [comment]
the sipplet from my private (work) project:

```
-- на боевых серверах хорошо бы запретить разварачивать (апгрейдить)
-- PRE-ревизии. этот скрипт показывает как ДБА может это сделать.

--begin;

set local role to "postgres";
set local search_path to public;

alter table face_pay_revision.face_pay_var_revision
        add constraint pre_revision_prevention_check
        check (revision !~~* 'PRE-%');

--commit;

-- vi:ts=4:sw=4:et
```

**Comment by polymorphm** [comment]
fixed in a4735b780eeba889accbe1d4f239e5a9ac3b62ea

and cherry picked in ``pg-make-schemas-8.0.1`` release
